### PR TITLE
Remove redundant code.

### PR DESCRIPTION
**Pull request by wayilau**
Signed-off-by: yunliliu <liuweiyi@cmss.chinamobile.com>

As i see the viper api. i think code redundant. so remove it .

### CODE REVIEW

**Comment by wayilau** [comment]
@stonezdj  please help me review this?

File: src/cmd/exporter/main.go [hunk 1]
```diff
@@ -19,7 +19,6 @@ import (
 func main() {
 	viper.AutomaticEnv()
```
Comment (steven-zou) [inline src/cmd/exporter/main.go:20]: Remove this line

Comment (wayilau) [reply src/cmd/exporter/main.go]: done, @steven-zou

**Comment by wayilau** [comment]
@bitsf please help me review this.

**Review by cndoit18 — APPROVED** [review]

**Comment by wayilau** [comment]
It's stilll nees another approving.

**Review by steven-zou — APPROVED** [review]
LGTM

**Comment by wy65701436** [comment]
@wayilau please help to rebase code to resolve the codeql issue, then let me merge it.

**Review by stonezdj — APPROVED** [review]

**Comment by stonezdj** [comment]
Please rebase the current branch with the main branch

**Comment by wy65701436** [comment]
@wayilau please help to rebase, let's me merge it once CI pass.

**Comment by wy65701436** [comment]
@wayilau do you still work on this? Rebase with latest could resove the CI failure, then I can merge it.

**Pull request closed without merging**